docs/src/api/class-page.md

```diff
@@ -989,6 +989,9 @@ Examples:
 const searchValue = await page.$eval('#search', el => el.value);
 const preloadHref = await page.$eval('link[rel=preload]', el => el.href);
 const html = await page.$eval('.main-container', (e, suffix) => e.outerHTML + suffix, 'hello');
+// In TypeScript, it is necessary to add an explicit type hint if you are not using a simple tag selector
```

Comment:
I'd prefer these examples to use the same selector as above. Like, `select#choose-options` or `link[rel=preload]` as above. I'd also skip the "using a simple tag selector" part since it just works. WDYT?

Reply:
Updated! I removed the simple tag example. The intent was to to show you don't always need it; I don't want people to see the type-hint docs and assume they always need it (and then potentially also give wrong hints). However, I suppose including the simple case, doesn't help communicate that unless you're already thinking about it.